EIPS/eip-1.md

```diff
@@ -185,6 +187,10 @@ The `created` header records the date that the EIP was assigned a number. Both h
 
 EIPs may have a `requires` header, indicating the EIP numbers that this EIP depends on.
 
+#### `resuscitatable` header
+
+EIPs may have a `resuscitatable` header. If this is `true`, then any PR that changes the `status` from `Stagnant` to any other state only requires editor approval.
```

Comment:
Unless I'm missing the point of this field, this description doesn't do it justice. I assume this is to support adopting stagnant EIPs and changing their author field? I think you should mention exactly what this lets non-authors do.

Changing from `Stagnant` to `Withdrawn` also seems weird without author approval, so I've added "non-final state".

```suggestion
EIPs may have a `resuscitatable` header. If this is `true`, then any PR changing `status` from `Stagnant` to a non-final state no longer requires author approval (instead only requiring editor approval.) Marking a proposal as `resuscitatable` allows _anyone_ to edit the author list and take over the proposal should it be abandoned.
```